src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1041,7 +975,7 @@ bool code_contractst::enforce_contract(const std::string &fun_to_enforce)
   auto nexist_old_function = goto_functions.function_map.find(original);
   INVARIANT(
     nexist_old_function == goto_functions.function_map.end(),
-    "There should be no function called " + fun_to_enforce +
+    "There should be no function called " + id2string(function) +
```

Comment:
Should this be `function` (which is used above in several places), or `id2string(function)`?

Reply:
It should be `id2string(function)` to use the correct overloaded operator +.

Reply:
Oh right, thanks!